### ISSUE TITLE
non-zero current in the off state

### ISSUE THREAD

**Issue by gogo10** [openshwprojects/OpenBK7231T_App#781]
**Describe the bug**
Non-zero current in the off state.

**Firmware:**
- Chip/model: BK7231N 
- Energy meter BL0942

![Bez názvu](https://user-images.githubusercontent.com/46536511/231377288-c48cafa5-a2a8-49a6-9b78-c872f2258443.png)

I don't know if this is a normal state or some error, please advise.
Thanks.

**Comment by openshwprojects** [comment]
Hello, was the device calibrated?

We could force zero value in off...

**Comment by gogo10** [comment]
Yes, it is calibrated, with a load of 60W it shows the correct values.

**Comment by openshwprojects** [comment]
I will add a flag to force-zero results when relays are off.

**Comment by gogo10** [comment]
Thank you.

**Comment by gogo10** [comment]
I'm sorry, it doesn't work properly, even when switched on and loaded with a 60W bulb, zero values are displayed.

**Comment by openshwprojects** [comment]
What is the configuration of your device?

**Comment by openshwprojects** [comment]
or which flag does not work properly? 25 or 38?

**Comment by gogo10** [comment]
I just wanted to write if the problem is not that I have a bistable relay and in the configuration it is controlled as a bridge?

**Comment by openshwprojects** [comment]
This changes much. The current flag only checks relays. So what is your current configuration? Pins and drivers etc

**Comment by openshwprojects** [comment]
Ok, can you check now, once the build is finished (wait 10 minutes for Github to process release)
